docs/standard/native-interop/tutorial-comwrappers.md

```diff
@@ -384,6 +384,58 @@ Since your `ComWrapper` subclass was designed to support `CreateObjectFlags.Uniq
 (rcw as IDisposable)?.Dispose();
 ```
 
+## COM Activation with `ComWrappers`
+
+The creation of COM objects is typically performed via COM Activation &ndash; a complex scenario outside the scope of this document. In order to provide a conceptual pattern to follow we introduce the [`CoCreateInstance()`][api_cocreateinstance] API, used for COM Activation, and illustrate how it can be used with `ComWrappers`.
+
+Assume you have the following C# code in your application. The below example uses `CoCreateInstance()` to activate a COM class and the built-in COM interop system to marshal the COM instance to the appropriate interface. Note the use of `typeof(I).GUID` is limited to an assert and is a case of using Reflection which can impact if the code is NativeAOT friendly.
```

Comment:
Reflection vs. reflection?
NativeAOT vs. Native AOT?
Both are in this doc.

Also, hyphen!

Reply:
```suggestion
Assume you have the following C# code in your application. The below example uses `CoCreateInstance()` to activate a COM class and the built-in COM interop system to marshal the COM instance to the appropriate interface. Note the use of `typeof(I).GUID` is limited to an assert and is a case of using reflection which can impact if the code is NativeAOT-friendly.
```